src/schema/v2/sale/index.ts

```diff
@@ -349,6 +361,24 @@ export const SaleType = new GraphQLObjectType<any, ResolverContext>({
         type: GraphQLBoolean,
         resolve: ({ auction_state }) => auction_state === "closed",
       },
+      isLotsClosing: {
+        type: GraphQLBoolean,
```

Comment:
_This_ is what should be a non-null boolean 😅

Reply:
Yes good call!